sites/geohub/src/components/util/stac/StacApiExplorer.svelte

```diff
@@ -869,10 +908,12 @@
 												{#each assetList as assetName}
 													{@const asset = feature.assets[assetName]}
 													<option value={assetName}>{asset.title ? asset.title : assetName}</option>
+													<option value={assetName}>{asset.title ? asset.title : assetName}</option>
 												{/each}
 											</select>
 										</div>
 									</div>
+									<div slot="help">{selectedTool.inputs.bands[index].description}</div>
```

Comment:
Please use `FieldControl`. 

Please read documentatiion
https://svelte-components.undpgeohub.org/?path=/docs/components-fieldcontrol--docs

To enable help popup, set `showHelp=true`. If `showHelpPopup` is true, help is shown as popup, if false, help message is shown. Use `help` slot for help message text.

If there is a component, please use it instead of using your own way. All style will become unified design